docs/guides/akamai/get-started/parent-child-accounts/index.md

```diff
@@ -0,0 +1,154 @@
+- This permission is available only for parent accounts.
+- Parent account users with limited access have this option disabled by default. They need to ask a parent account admin to enable this option for them.
+- It enables access to **all** child accounts the parent account has a contractual parent/child relationship with.
+
+To enable the access:
+1. Log in to the [Cloud Manager](https://cloud.linode.com/).
+1. In the main menu, go to **Account**.
+1. In the **User & Grants** tab, next to the name of a parent account user with limited access, click **...** > **User Permissions**.
+1. In the **General Permissions** section, switch the **Enable child account access** option on. Click **Save**.
+
+### Switch accounts
+
+When switching to a child account, you become the child account parent user. By default, you have:
+- Read/write permission for billing. This is the only permission that can’t be modified even by a child account admin.
+- Read-only permission to all asset instances existing at the time of parent/child relationship creation, such as linodes, load balancers, longview clients. This allows the child account parent user to see existing child accounts’ assets, but not create, update, or delete them.
+- No access to asset instances created after the creation of the parent/child relationship. The child account parent user needs to ask a child account admin to grant them access to new instances if necessary.
+- The [global permissions](/docs/products/platform/accounts/guides/user-permissions/) set off.
+
+A child account admin can change the child account parent user’s default permissions and expand or restrict its access to the child account, except for billing. If the child account admin provides the child account parent user with full access to the child account, the child account parent user can fully manage the account, including its users.
+
+To switch accounts:
+1. Log in to the [Cloud Manager](https://cloud.linode.com/).
+1. In the top corner, click the name of your account.
+
+    ![Parent account switch](parent_switch.png)
+
+1. Click the **Switch account** button. If you don’t see this button, it means you’re a parent account user with limited access and you need to ask the parent account admin to provide you the access to child accounts or your account is not involved in the parent/child relationship.
+
+    ![Account switch button](switch_account.png)
+
+1. Select an account from the list you want to switch to.
+Now you act as the child account parent user. Note that you can click the **switch back to your account** link to switch back.
+
+    ![switch back option](switch_back.png)
+
+### Delete a parent account
+A parent account can be deleted only if all parent/child relationships are contractually removed from your parent account, meaning you don’t have active contracts with end customers.
+
+Once this condition is met, to close an account:
```

Comment:
Do we cover how to meet this condition anywhere? Are we concerned that customers will ask? (I figure this is fine for now, but I just wanted to call this out to be safe.)